documentation/stackoverflow.workspace.adoc

```diff
@@ -267,8 +401,16 @@ WHERE freq > 2
 RETURN u1, u2, apoc.create.vRelationship(u1,'OCCURRED',{freq:freq},u2) as rel
 ----
 
-== Further Reading
+In the next section, you will find more resources to continue learning about Neo4j.
+
+== Next steps
+
+Congratulations on completing this guide!
+
+You can continue your Neo4j learning journey with these resources:
+
+https://neo4j.com/graphacademy[Neo4j GraphAcademy - completely free online courses^]
+
+https://neo4j.com/videos/exploring-stackoverflow-data-with-michael-hunger-twitch-stream/[Exploring StackOverflow data with Michael Hunger – Twitch stream^]
 
-* https://neo4j.com/labs/apoc/4.1/import/load-json/#load-json-examples-stackoverflow[Import from StackOverflow API^]
-* https://neo4j.com/videos/exploring-stackoverflow-data-with-michael-hunger-twitch-stream/[Exploring StackOverflow data with Michael Hunger – Twitch stream^]
-* https://neo4j.com/blog/import-10m-stack-overflow-questions/[Import 10M Stack Overflow Questionsinto Neo4j In Just 3 Minutes^]
+https://neo4j.com/blog/import-10m-stack-overflow-questions/[Import 10M Stack Overflow Questionsinto Neo4j In Just 3 Minutes^]
```

Comment:
there is also the tagoverflow post https://towardsdatascience.com/tagoverflow-correlating-tags-in-stackoverflow-66e2b0e1117b